batch7crud-roll7/src/main/java/com/lftechnology/batch7crud/dao/StudentDataAccess.java

```diff
@@ -0,0 +1,95 @@
+package com.lftechnology.batch7crud.dao;
+
+import com.lftechnology.batch7crud.db.DbUtilities;
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.model.Student;
+
+import javax.naming.NamingException;
+import javax.xml.crypto.Data;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'javax.xml.crypto.Data'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)